app/assets/javascripts/angular/controllers/frame_ctrl.js

```diff
@@ -92,7 +92,6 @@ angular.module("Prometheus.controllers").controller('FrameCtrl', ["$scope",
         return o.key;
       }
     }).join("&");
-    $scope.frame.escapedURL = VariableInterpolator(parser.href, $scope.vars);
```

Comment:
I got rid of this escapedURL monkey business in favor of only interpolating into the URL when we make the actual iframe.